content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -1,70 +1,172 @@
+route is selected in the :guilabel:`Inventory` tab of the product form.
+
+Then, create a reordering rule to automate replenishment by clicking the :guilabel:`Reordering
+Rules` smart button.
+
+Click :guilabel:`New`, and set:
+
+- :guilabel:`Location`: the stock location of the retail store. For example, `SHOP/Stock`.
+- :guilabel:`Route`: :guilabel:`X: Supply Product from Y`.
+- :guilabel:`Min Quantity` and :guilabel:`Max Quantity` to trigger automatic stock transfers when
+  inventory falls below the set threshold.
+
+.. seealso::
+   :doc:`reordering_rules`
+
+.. example::
+   A :ref:`0/0 reordering rule <inventory/warehouses_storage/zero-zero>` to replenish the shop's
+   warehouse is created, with the :guilabel:`Location` set to `SHOP/Stock`, and the
+   :guilabel:`Route` set to :guilabel:`Store: Resupply from YourCompany`.
+
+   .. image:: resupply_warehouses/reordering-rule.png
+      :align: center
+      :alt: Show reordering rule configurations.
+
+.. _inventory/warehouses_storage/resupply-workflow:
+
+Replenish one warehouse from another
+====================================
+
+After completing the setup, trigger replenishment using one of several methods, such as:
+
+- Navigate to the product form of the product that is resupplied from another warehouse.
+
+  Click the :guilabel:`Replenish` button on the top-left of the product page. In the pop-up window,
+  set the warehouse to the retail shop, (e.g. `Store`), and click :guilabel:`Confirm`.
+
+  .. image:: resupply_warehouses/replenish.png
+     :align: center
+     :alt: Replenish pop-up window on the product form.
+
+- Create a quotation, and in the :guilabel:`Other Info` tab, set the :guilabel:`Warehouse` to the
+  retail shop (e.g. `Store`), when selling the product makes the on-hand quantity of the product go
+  below the minimum set on the reordering rule.
+
+  .. image:: resupply_warehouses/warehouse-field.png
+     :align: center
+     :alt: Create a quote at the store.
+
+Once triggered, Odoo creates two transfers: One is a *delivery order* from the central, supplying
+warehouse, which contains all the necessary products to the store, and the second is a *receipt* at
+the shop, from the main warehouse.
+
+While in transit, the product is located at `Physical Locations/Inter-warehouse transit`.
+
+.. example::
+   A sales order for the product at the shop is created. To replenish the product at the shop and
+   ship it from there, Odoo generates a delivery order from the central warehouse's stock,
+   `WH/Stock` to the shop's warehouse `SHOP/Stock`. While the products are traveling between
+   warehouses, they are in `Physical Locations/Inter-warehouse transit`.
+
+   The final delivery order is from the shop to the customer's delivery address, and is not
+   pertinent to the workflow in this guide.
 
-.. image:: resupply_warehouses/second-warehouse-stock-receipt.png
+.. image:: resupply_warehouses/transfers.png
```

Comment:
Seems like this image should be indented to be included in the example admonition :)
```suggestion
   .. image:: resupply_warehouses/transfers.png
      :alt: Show shipments from warehouse to store.
```